docs/UPGRADING.rst

```diff
@@ -0,0 +1,20 @@
+Upgrading HaaS
+==============
+
+This file describes the procedure for upgrading HaaS to a new version.
+
+1. Read the release notes, as they may contain version-specific information.
+2. Stop the HaaS services (webserver and HaaS network daemon).
+3. Download and install the new version of HaaS::
+
+     $ git clone https://github.com/cci-moc/haas
+     $ python setup.py install
```

Comment:
`setup.py` doesn't have the `--upgrade` option (only pip does), so if we wanted to change it we'd have to change it to pip. The `--upgrade` option won't have any different behavior when used to install a package from a file path; it's only useful when specifying packages to be fetched.

As far as reasons for preference, the only think I can think of is the case where pip isn't present on the system, which is not unheard of.

The reason I wrote it this way is because it's consistent with what's already in `INSTALL.rst`.